apps/snitch_core/test/data/model/payment/payment_test.exs

```diff
@@ -37,11 +34,21 @@ defmodule Snitch.Data.Model.PaymentTest do
       |> get_naive_date_time()
 
     payment_date_count =
-      Model.Payment.get_payment_count_by_date(payment.inserted_at, next_date) |> List.first()
+      Payment.get_payment_count_by_date(payment.inserted_at, next_date) |> List.first()
 
     assert %Money{} = payment_date_count.count
   end
 
+  test "get_all/0 return all payments", %{ccd: cod, chk: check, hpm: hosted_payment} do
+    returned_hosted_payments = Payment.get_all()
```

Comment:
variable name should be `returned_payments` and not `returned_hosted_payments`, since it returns payments